Tests/Bricks/CollectionBrickTests.swift

```diff
@@ -129,6 +129,43 @@ class CollectionBrickTests: XCTestCase {
         XCTAssertEqual(delegate.count, 1, "The delegate should have only been called 1 time for the updated height of the CollectionBrick")
     }
 
+    func testThatCollectionViewZeroFrameHeight() {
+        brickView.registerBrickClass(CollectionBrick.self)
+
+        let collectionSection = BrickSection("CollectionSection", bricks: [
+            DummyBrick("1", height: .Auto(estimate: .Fixed(size: 1000))),
+            DummyBrick("2", height: .Auto(estimate: .Fixed(size: 1000))),
+            DummyBrick("3", height: .Auto(estimate: .Fixed(size: 1000))),
+            DummyBrick("4", height: .Auto(estimate: .Fixed(size: 1000)))
+            ])
+
+        let section = BrickSection("RootSection", bricks: [
+            CollectionBrick("Collection1", dataSource: CollectionBrickCellModel(section: collectionSection, configureHandler: { cell in
```

Comment:
You can use the `brickTypes: [DummyBrick.self])` in the initializer to make the constructor easier readable